static/styles/dark_theme.css

```diff
@@ -577,6 +577,10 @@
         }
     }
 
+    .message_row .date_row span {
+        opacity: 0.75;
+    }
```

Comment:
I removed this likely stale hunk, which seemed to introduce an accidental 0.75 squared opacity (and made this color different from the one selected via `color-date` below).